app-text/texlive/texlive-2024.recipe

```diff
@@ -196,8 +195,6 @@ GLOBAL_WRITABLE_FILES="
 
 PATCH() {
 	sed -i "s|\`kpsewhich -var-value=TEXMFROOT\`|\"$dataDir/texlive\"|" $sourceDir/tlpkg/TeXLive/TLUtils.pm
-	sed -i "s|\`kpsewhich -var-value=FC_CACHEDIR\`|\"$sharedStateDir/texlive/fonts/cache\"|" $sourceDir3/tlpkg/tlpostcode/xetex.pl
-	sed -i "s|\`kpsewhich -var-value=FONTCONFIG_PATH\`|\"$sharedStateDir/texlive/fonts/conf\"|" $sourceDir3/tlpkg/tlpostcode/xetex.pl
```

Comment:
This sed change doesn't apply any more, because they changed the involved variable names, see https://tug.org/svn/texlive?revision=70088&view=revision

It doesn't seem to be needed any more though (the package builds successfully), presumably because of exactly that change.

note: I did not actually test XeTeX.